modules/compliance-unapplying.adoc

```diff
@@ -12,7 +12,7 @@ It might be required to unapply a remediation that was previously applied.
 +
 [source,terminal]
 ----
-$ oc patch complianceremediations/<scan_name>-sysctl-net-ipv4-conf-all-accept-redirects -p '{"spec":{"apply":false}}' --type=merge
+$ oc patch -n openshift-compliance complianceremediations/rhcos4-e8-sysctl-net-ipv4-conf-all-accept-redirects -p '{"spec":{"apply":false}}' --type=merge
```

Comment:
same as above, should keep it as original or use command below instead:
`$ oc patch -n openshift-compliance complianceremediations/rhcos4-moderate-worker-sysctl-net-ipv4-conf-all-accept-redirects --patch '{"spec":{"apply":false}}' --type=merge`